common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal.cpp

```diff
@@ -1374,6 +1370,7 @@ bool mon_wlan_hal_dwpal::process_dwpal_nl_event(struct nl_msg *msg)
             }
             LOG(DEBUG) << "Processing results for BSSID:" << results->channel_scan_results.bssid;
             event_queue_push(event, results);
+            return true;
         } else {
             LOG(ERROR) << "channel scan results dump received with unexpected seq number";
             return false;
```

Comment:
You can get rid of the nesting here too, although I'm not sure I understand why did you add the`return true` here.

Reply:
Because of the nesting I wanted the correct result to return `true`
So i removed the nesting, and now it goes to `break` which returns `true `